### PR TITLE
Feature/component task

### PR DESCRIPTION
**Pull request by RomanKondr**

### CODE REVIEW

File: src/components/ui/task.tsx [hunk 1]
```diff
@@ -16,10 +16,20 @@ import Animated, {
 import { Ionicons } from '@expo/vector-icons';
 import { cn } from '../../lib/utils';
 
+const DEFAULT_HEIGHT = 70;
+const MAX_BOX_WIDTH = 120;
+const SLIDE_THRESHOLD = 15;
```
Comment (Lomsss) [inline src/components/ui/task.tsx:21]: Лють

File: src/components/ui/task.tsx [hunk 2]
```diff
@@ -29,12 +39,20 @@ export interface Task {
...
+  item,
+  isActive,
+  isVisible,
+  isCompact,
+  onThreeDotsPress,
+  taskHeight = 160,
+  taskWidth = '100%',
+  compactHeight = 60,
+  compactWidth = '86%',
+}) => {
```
Comment (Lomsss) [inline src/components/ui/task.tsx]: Use native wind for styles

File: src/components/ui/task.tsx [hunk 3]
```diff
@@ -77,196 +104,207 @@ const TaskItem: React.FC<TaskItemProps> = ({ item, isActive, isVisible }) => {
...
-        </Animated.View>
-      </PanGestureHandler>
+        </View>
+      )}
 
-      <Animated.View style={DoneBoxStyle}>
-        <Ionicons name="checkmark" size={24} color="white" />
-      </Animated.View>
+      {!isCompact && (
+        <>
```
Comment (Lomsss) [inline src/components/ui/task.tsx]: why do you use `<>`?

**Review by Lomsss — CHANGES_REQUESTED** [review]

File: src/components/ui/task.tsx (hunk 3 above)
Comment (RomanKondr) [reply src/components/ui/task.tsx]: Used it because in other ways delete and done boxes will be under task, didn't find other way to replace it

**Pull request merged**